packages/core/contracts/common/interfaces/TransactionBatcher.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+
+// See deployed implementation here: https://etherscan.io/address/0x82458d1c812d7c930bb3229c9e159cbabd9aa8cb.
+abstract contract TransactionBatcher {
+    function batchSend(
+        address[] memory targets,
+        uint256[] memory values,
+        bytes[] memory datas
+    ) public virtual payable;
+}
```

Comment:
is there some kind of magic here? where is the implementation

Reply:
Updated the comment. This is the interface of a contract that was already deployed by someone else. I should've made that clearer.